molecule/plaintext-rhel-customrepo/prepare.yml

```diff
@@ -9,27 +9,23 @@
         ansible_distribution_major_version: 7
     - import_role:
         name: variables
-    - debug:
-        msg:
-          - "Confluent Client Repo Version -> {{ confluent_common_repository_redhat_release_version }}"
-          - "OS Family -> {{ ansible_os_family }}"
     - name: Create custom repo file
       copy:
         dest: "{{ custom_yum_repofile_filepath }}"
         content: |
           [Confluent.custom]
           async = 1
-          baseurl = "https://packages.confluent.io/rpm/{{confluent_repo_version}}"
+          baseurl = {{confluent_common_repository_baseurl}}/rpm/{{confluent_repo_version}}
           enabled = 1
           gpgcheck = 1
-          gpgkey = "https://packages.confluent.io/rpm/{{confluent_repo_version}}/archive.key"
+          gpgkey = {{confluent_common_repository_baseurl}}/rpm/{{confluent_repo_version}}/archive.key
           name = Custom Confluent repository
           [Confluent.clients.custom]
           async = 1
-          baseurl = "https://packages.confluent.io/clients/rpm/centos/{{confluent_common_repository_redhat_release_version}}/x86_64"
+          baseurl = https://packages.confluent.io/clients/rpm/centos/{{confluent_common_repository_redhat_release_version}}/x86_64
```

Comment:
Client side is available to public, right? 
lgtm

Reply:
yeah, the client's version don't change with patch, and we don't host clients in staging URLs for eg https://packages.confluent.io/clients/rpm/centos/7/x86_64/ there is no https://<staging-url>/clients/rpm/centos/7/x86_64/
But I am afraid this will also break in major release like 8.0